readme.md

```diff
@@ -7,128 +7,241 @@
+🚀 CI/CD ready out of the box (Tested in GitHub Actions CI - Please let us know if it just works(tm) in other CI systems)
 
-```
-pip uninstall llm-app-test 
-pip install llm-app-test==0.1.0b3.post3
-```
-#### Method 2: Force Reinstall (if Method 1 fails)
+💰 Cost-effective testing solution
+
+🔧 No infrastructure needed (Unless if you want to inject a custom LLM, please refer to the configuration page of the documentation for details)
+
+## Installation - Reading the rest of the readme first is strongly recommended before use
 
 ```
-pip install --force-reinstall llm-app-test==0.1.0b3.post3
-```
-#### Verification
-After rolling back, verify the installation:
-```
-import llm_app_test 
-print(llm_app_test.version) # Should show 0.1.0b3.post3
+pip install llm-app-test
 ```
 
-## What llm_app_test Does
+Please refer to the [documentation](https://shredmetal.github.io/llmtest/getting-started/installation/) for full instructions.
+
+## Testing Philosophy
+
+When integrating LLMs into your application, treat them as you would any closed-source third-party library (because there's a pretty good chance you can't actually change the LLM itself):
+
+1. Write tests for expected behaviour
+2. Focus on interface boundaries
+3. Test application-level functionality
+4. Maintain clear separation of concerns
+
+### ⚠️ Important Information on Understanding Responsibilities
+
+llm-app-test is built by engineers for engineers so that all of us have a tool to validate the behaviour of applications that have had an LLM stuffed into them. It is **NOT** a Data Science tool nor a replacement for model metrics used by Data Science teams to validate model suitability.
+
+#### Software Engineer's Role
+
+- Write tests for expected application behaviour (in the same way you would for any piece of production code)
+- Validate inputs and outputs
+- Ensure proper integration
+- Monitor system performance (if all of your llm_app_test tests were passing before, and they suddenly start failing, you might have a problem)
+- Escalate consistent failures to DS team (as this might indicate a fundamental problem with the model, or perhaps it might be worth seeking assistance with the `expected_behavior` prompt in the `assert_behavioral_match` function)
+
+#### Data Science Team's Role
+
+- Handle model-level issues
+- Address consistent test failures
+- Evaluate model suitability
+- Optimise model performance
+- Adjust prompts when needed
+
+### When to Escalate
+
+Escalate to your Data Science team when:
+
+1. Tests consistently fail despite correct implementation
+2. Model responses are consistently inappropriate
+3. Performance degradation is observed
+4. Pattern of failures indicates model-level issues
+
+### 🔍 What Makes This Different?
+
+This is an ENGINEERING tool, not a data science tool. The difference is crucial:
+
+Data Science Tools:
+- Test model performance
+- Evaluate model accuracy
+- Measure model metrics
+
+llm_app_test (Engineering Tool):
```

Comment:
llm-app-test

Reply:
Undone - standardise to llm_app_test because that's what the import statement is going to say. llm-app-test is only used for pip install because PyPI converts underscores to hyphens